controller/src/main/java/io/pravega/controller/server/eventProcessor/LocalController.java

```diff
@@ -199,7 +199,7 @@ public LocalController(ControllerService controller, boolean authorizationEnable
 
     @Override
     public CompletableFuture<Boolean> createStream(String scope, String streamName, final StreamConfiguration streamConfig) {
-        return this.controller.createStream(scope, streamName, streamConfig, System.currentTimeMillis(), requestIdGenerator.nextLong()).thenApply(x -> {
+        return this.controller.createStreamInternal(scope, streamName, streamConfig, System.currentTimeMillis(), requestIdGenerator.nextLong()).thenApply(x -> {
```

Comment:
This does not look right. `LocalController.createStream()` is also used for creating user streams.
What is needed is to add a new method to LocalController - `createStreamInternal` that internally invokes `this.controller.createStreamInternal` and keep the original createStream as is.

Reply:
Made the changes as suggested